### ISSUE TITLE
Script files missing?

### ISSUE THREAD

**Issue by pacmyc** [molobrakos/tellsticknet#29]
In the readme there are many examples involving scripts, but  I can't find the scripts anywhere?

**Comment by oklona** [comment]
No, the readme seems to be related to a previous version... If you look at a very old commit, for instance this: https://github.com/molobrakos/tellsticknet/tree/bba814cfc6e1f9b747eb5d45ca1371ab1ec719ec, you will find them. They seem to have been (more or less) incorporated into the main script.
